crates/ironrdp-dvc/src/lib.rs

```diff
@@ -98,21 +98,67 @@ pub fn encode_dvc_messages(
     Ok(res)
 }
 
-pub struct DynamicVirtualChannel {
+pub struct DynamicVirtualChannel<'a, T: DvcProcessor> {
```

Comment:
`DynamicVirtualChannel`/`DynamicVirtualChannelInternal` separation feels a bit confusing to me, what was the reason for such refactoring?

Reply:
Yeah I took a wrong turn somewhere there, fixed in 1fd4b38c34777fa52d94dd56e2a112efde4ae62e